api/api.ts

```diff
@@ -0,0 +1,66 @@
+import axios from 'axios';
+import { GameInfo, UserInfo, UserGames } from 'types/responseInterface';
+
+const END_POINT = 'http://localhost:8080';
+
+const getRequest = <T>(url: string) => {
+    try {
+        return axios.get<T>(`${END_POINT}/${url}`);
+    } catch (e) {
+        console.error(e);
+        return e;
+    }
+};
+
+const deleteRequest = (url: string) => {
+    try {
+        const response = axios.delete(`${END_POINT}/${url}`);
+        console.log(response);
+        // return response;
+    } catch (e) {
+        console.error(e);
+    }
+};

```

Comment:
제가 생각했던 getRequest, deleteRequest, postRequest의 경우 나중에 axios가 아닌 fetch로 변경한다던가 할 경우에 전체 코드 변경하지 않고 해당 부분만 변경해서 적용하는 용으로 쓰고 싶었습니다.
```suggestion
const deleteRequest = (url: string) => {
    return axios.delete(`${END_POINT}/${url}`);
};

```
그래서 이렇게 그냥 처리하고 각각 상황에 따른 처리는 사용하는 쪽에서 처리하려고 했습니다.

Reply:
앗 이해했습니다! 해당 모듈 사용하는 부분에서 사용한다는 의미군요! 감사합니다 🙏